src/services/consent/utils/index.js

```diff
@@ -0,0 +1,112 @@
+const createConsentFilterQuery = (...status) => {
+	const currentDate = new Date();
+	const secondConsentSwitchDate = new Date();
+	secondConsentSwitchDate.setFullYear(currentDate.getFullYear() - 16); // TODO: get age from default.conf
+	const firstConsentSwitchDate = new Date();
+	firstConsentSwitchDate.setFullYear(currentDate.getFullYear() - 14);
+
+	const requiredConsents = {
+		privacyConsent: true,
+		termsOfUseConsent: true,
+	};
+
+	const missingConsents = {
+		$or: [
+			{ privacyConsent: false },
+			{ termsOfUseConsent: false },
+		],
+	};
+
+	const orConditions = status.reduce((query, status) => {
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: ['status' is already declared in the upper scope.](https://app.codacy.com/manual/schul-cloud/schulcloud-server/pullRequest?prid=5276619)